imports/ui/dateHelpers.coffee

```diff
@@ -0,0 +1,12 @@
+dateStringToDate = (dateString, delim) ->
```

Comment:
Could you replace the dateHelpers module with moment.js? Calls like `moment(dateString, "MM/DD/YYYY")` or `moment(date).format("MM/DD/YYYY")` are preferable since it possible to tell what kind of format is being used without referring to a function definition in another file.